### PR TITLE
Add support for Python 3.8

### PR DESCRIPTION
**Pull request by 921kiyo**
## Description

Kedro 0.16.* has been released, which supports Python 3.8. So we are also supporting Python 3.8 for Kedro-Viz plugins. 

## Development notes
Modified the e2e tests 

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: package/features/environment.py [hunk 1]
```diff
@@ -96,7 +104,7 @@ def call(cmd, verbose=False):
     context.temp_dir = Path(tempfile.mkdtemp())
 
 
-def after_scenario(context, feature):
```
Comment (921kiyo) [inline package/features/environment.py:99]: `feature` arg was never used, so I removed it.

**Comment by mzjp2** [comment]
We need to pin `kedro>=0.16` in `requirements.txt`, right?

cc @DmitriiDeriabinQB

**Comment by DmitriiDeriabinQB** [comment]
> We need to pin `kedro>=0.16` in `requirements.txt`, right?
> 
> cc @DmitriiDeriabinQB

No, I don't think so. Earlier versions of Kedro won't install anyways if you're running Python 3.8.

File: package/features/environment.py [hunk 2]
```diff
@@ -32,17 +32,25 @@
...
+import sys
 import tempfile
 from pathlib import Path
 
 from features.steps.sh_run import run
 from features.steps.util import create_new_venv
 
 
-def before_scenario(context, feature):  # pylint: disable=unused-argument
+def _should_exclude_scenario(scenario):
```
Comment (DmitriiDeriabinQB) [inline package/features/environment.py:43]: Nice! 👍

**Comment by DmitriiDeriabinQB** [comment]
> LGTM! Although @DmitriiDeriabinQB suggested that maybe we should consider dropping pre-15 version of Kedro.

Yeah, I think that's the plan for the next release (`3.4` or `4.0`).

**Pull request merged**